common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -103,12 +109,49 @@ like_elekks_trigger = {
 		has_innovation = innovation_elephantry
 	}
 }
+like_dragonhawk_trigger = {
+	OR = {
+		this = culture:blood_elf
+		this = culture:high_elf
```

Comment:
You shouldn't forget about steppes. Many cavalry units like kodos don't have bonuses there.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/157848438-89c869cb-4be9-4b45-a2b0-b76a8c4da1ad.png)

</details>

Reply:
Also, I noticed basic aerial unit don't have steppes bonus.